cluster/plan.go

```diff
@@ -1286,3 +1293,26 @@ func (c *Cluster) IsCRIDockerdEnabled() bool {
 	}
 	return false
 }
+
+func (c *Cluster) multipleCIDRsConfigured() bool {
+	if c == nil {
+		logrus.Debug("multipleCIDRsConfigured: Returning false, cluster object is nil")
+		return false
+	}
```

Comment:
Hold on, if `c == nil` we would never reach this code, right? This check is probably not needed

Reply:
It is aligned with the other function that checks for c being `nil` (https://github.com/rancher/rke/blob/release/v1.4/cluster/plan.go#L1280), that's why I added it.

Reply:
Same problem there. It is impossible to reach that code because the struct does not exist, so we could never reach its methods

Reply:
I have just run a quick test and there might be a way to reach the method of a struct even if the struct is nil. Let me quickly check this tomorrow, I might be wrong after all

Reply:
@manuelbuil I can confirm that calling a method on a nil struct is fine, it will only panic if you try to access it in some way, like line 1302. Source: https://stackoverflow.com/questions/42238624/calling-a-method-on-a-nil-struct-pointer-doesnt-panic-why-not